### PR TITLE
feat(sengrid.md): Update docs for SendGrid Connector

### PR DESCRIPTION
**Pull request by Oleksiivanov**
## What is the purpose of the change

Update [documentation](https://docs.camunda.io/docs/next/components/integration-framework/connectors/out-of-the-box-connectors/sendgrid/) so it reflects the new element template.

## Are there related marketing activities

https://github.com/camunda/team-connectors/issues/138

## When should this change go live?

-

## PR Checklist

- [ ] My changes apply to an already released version, and I have added them to the relevant `/versioned_docs` directory, or they are not for an already released version.
- [ ] My changes apply to future versions, and I have added them to the main `/docs` directory, or they are not for future versions.
- [ ] My changes require an [Engineering review](https://github.com/camunda/camunda-platform-docs/blob/main/howtos/documentation-guidelines.md#review-process), and I've assigned an engineering manager or tech lead as a reviewer, or my changes do not require an Engineering review.
- [ ] My changes require a [technical writer review](https://github.com/camunda/camunda-platform-docs/blob/main/howtos/documentation-guidelines.md#review-process), and I've assigned @christinaausley as a reviewer, or my changes do not require a technical writer review.

### CODE REVIEW

File: docs/components/integration-framework/connectors/out-of-the-box-connectors/sendgrid.md [hunk 1]
```diff
@@ -100,12 +101,13 @@ To make the **SendGrid Email Template Connector** executable, fill out all the m
 3. Set **Sender Email** to `jane-doe@camunda.com` (or the [sender identity](#create-a-sender-identity) you configured above).
 4. Set **Receiver Name** to `Your Name`.
 5. Set **Receiver Email** to `Your email address`.
-6. Log in to your SendGrid account and navigate to [the Dynamic Template you created](#configure-a-dynamic-template).
-7. Copy the id of the template and paste it in the **Template ID field**.
+6. Choose **Using dynamic template** in **Compose Email** section.
+7. Log in to your SendGrid account and navigate to [the Dynamic Template you created](#configure-a-dynamic-template).
+8. Copy the id of the template and paste it in the **Template ID field**.
```
Comment (igpetrov) [inline docs/components/integration-framework/connectors/out-of-the-box-connectors/sendgrid.md:106]:
```suggestion
8. Copy the ID of the template and paste it in the **Template ID field**.
```

**Comment by christinaausley** [comment]
@Oleksiivanov Note that @akeller is a bit tied up with responsibilities surrounding the [DevRel restructure](https://camunda.slack.com/archives/C0PLU0Y12/p1666623814293859) -- if you'd like a technical review this week feel free to add me as a reviewer 👍

**Review by christinaausley — DISMISSED** [review]
Very minor adjustments made in my latest commit 👍

**Review by igpetrov — APPROVED** [review]
Thank you.

**Pull request merged**